src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.LeakDetection/CandidateFileEntry.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+
+namespace Microsoft.DotNet.SourceBuild.Tasks.LeakDetection
+{
+    internal class CandidateFileEntry
+    {
+    public string ExtractedPath { get; set; }
+    public string FullPath { get; set; }
```

Comment:
If I understand this correctly the fullPath is a path that includes the tarball name.  This makes sense but the member name is a bit deceptive in a way because it is not an actual file system path.  I wonder if there is a better name to call this out?  Maybe DisplayPath would be better as it indicates the intended usage?

Along these lines the ExtractedPath is a full path as well right?  That also adds to the confusion on what FullPath means.

Reply:
I agree that DisplayPath makes more sense than FullPath.

Extracted path uses a temporary directory. One example is below:

```
ExtractedPath: /tmp/4swt1hgf.pkw/FSharp.Core.8.0.100-beta.23371.8/lib/netstandard2.1/cs/FSharp.Core.resources.dll
FullPath: /repos/dotnet/artifacts/x64/Release/Private.SourceBuilt.Artifacts.8.0.100-preview.7.23376.1.fedora.38-x64.tar.gz/Microsoft.FSharp.Compiler.12.8.0-beta.final.nupkg/contentFiles/Shipping/FSharp.Core.8.0.100-beta.23371.8.nupkg/lib/netstandard2.1/cs/FSharp.Core.resources.dll
RootPath: /repos/dotnet/artifacts/x64/Release
```

Reply:
Changed FullPath to RootPath: https://github.com/dotnet/installer/pull/17438/commits/b99b930bf2726d24488910226e0fac3f3575e515